packages/authenticator/amplify_authenticator/lib/src/l10n/generated/button_localizations_en.dart

```diff
@@ -48,6 +51,21 @@ class AuthenticatorButtonLocalizationsEn
   @override
   String get skip => 'Skip';
 
+  @override
+  String get openTotpApp => 'Open Authenticator App';
+
+  @override
+  String get totpManualKey => 'Manual Key Entry';
+
+  @override
+  String get totpQrCode => 'Scan QR Code';
+
+  @override
+  String get totpStepperAppSetup => 'Step up Authenticator App';
```

Comment:
Is this supposed to be `Set up`?

```suggestion
  String get totpStepperAppSetup => 'Set up Authenticator App';
```